### PR TITLE
DPDTimeout Multiple Radius Server for VPN Gateways

### PR DESCRIPTION
**Pull request by henry416**
Adding DPDTimeout and Multiple Radius Servers for VPN Gateways
DPDTimeout is an integer field in connection resources.
Multiple Radius is a subobject containing radius sever address/score/secret in gateway resources.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualNetworkGateway.json [hunk 1]
```diff
@@ -2459,6 +2459,24 @@
       ],
       "description": "VPN client revoked certificate of virtual network gateway."
     },
+    "RadiusServer": {
```
Comment (lirenhe) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualNetworkGateway.json]: Is this a breaking change as you add new property to the response?

File: specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualWan.json [hunk 2]
```diff
@@ -3745,6 +3745,11 @@
           "format": "int32",
           "description": "Routing weight for vpn connection."
         },
+        "dpdTimeoutSeconds": {
```
Comment (lirenhe) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualWan.json]: Same above

**Comment by number213** [comment]
Please submit this change against `network-march-release`

**Review by anton-evseev — CHANGES_REQUESTED** [review]
Please submit this change against `network-march-release`

File: specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualNetworkGateway.json (hunk 1 above)
Comment (henry416) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualNetworkGateway.json]: No.

File: specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualWan.json (hunk 2 above)
Comment (henry416) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualWan.json]: No.

**Comment by henry416** [comment]
@lirenhe @number213 Created new PR targeting network-march-release
https://github.com/Azure/azure-rest-api-specs/pull/8707

**Pull request closed without merging**